pkg/printers/internalversion/printers.go

```diff
@@ -2779,6 +2787,41 @@ func printClusterCIDRList(list *networking.ClusterCIDRList, options printers.Gen
 	return rows, nil
 }
 
+func printIPAddress(obj *networking.IPAddress, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+
+	parentRefName := "<none>"
+	if obj.Spec.ParentRef != nil {
+		gr := schema.GroupResource{
+			Group:    obj.Spec.ParentRef.Group,
+			Resource: obj.Spec.ParentRef.Resource,
+		}
+		parentRefName = strings.ToLower(gr.String())
+		if obj.Spec.ParentRef.Namespace != "" {
+			parentRefName += "/" + obj.Spec.ParentRef.Namespace
```

Comment:
Can an IPAddress be owned by a non namespaced object?

Reply:
I never thought about that, main use cases are Services and Gateway API, @thockin WDYT

Reply:
I think it should be owned by a non-namespace object, with the new multi-network group APIS I see that some APIs that may require IP addresses are non-namespaced

Reply:
I can see that I also implemented it this way, in validation I added a comment that namespace is optional.

This is also allowd, non-namespaced -> non-namespaced and namespaced -> non-namespaced are ok